### PR TITLE
Added Air Century SA, Jetair Caribbean. Invalidated Ygnus Air.

### PR DESCRIPTION
**Pull request by thepermanenturl**
Couldn't find validity dates so put today's dates.

Added Air Century SA, Jetair Caribbean. Invalidated Ygnus Air.

### CODE REVIEW

File: opentraveldata/optd_airline_best_known_so_far.csv [hunk 1]
```diff
@@ -71,6 +71,7 @@ air-air-canada-v1^^1965-01-01^^ACA^AC^14^Air Canada^^^Member^^https://en.wikiped
 air-air-caraibes-v1^^2000-07-01^^FWI^TX^427^Air Caraibes^^^^^https://en.wikipedia.org/wiki/Air_Cara%C3%AFbes^en|Air Caraibes|^^air-air-caraibes^1^
 air-air-cargo-carriers-v2^^2015-01-01^^SNC^2Q^0^Air Cargo Carriers^^^^C^https://en.wikipedia.org/wiki/Air_Cargo_Carriers^en|Air Cargo Carriers|^^air-air-cargo-carriers^2^
 air-air-carnival-v1^^2016-07-18^^^2S^0^Air Carnival^^^^^https://en.wikipedia.org/wiki/Air_Carnival^en|Air Carnival|^CJB^air-air-carnival^1^
+air-air-century-sa-v1^^2022-11-15^^CEY^Y2^Air Century, S.A.^^^^^https://en.wikipedia.org/wiki/Air_Century^^^^1^
```
Comment (alexprengere) [inline opentraveldata/optd_airline_best_known_so_far.csv:74]: I think there is a missing caret on that line, right before the airline name

File: opentraveldata/optd_airline_best_known_so_far.csv [hunk 2]
```diff
@@ -558,6 +559,7 @@ air-jazz-aviation-v1^^^^JZA^QK^983^Jazz Aviation^Jazz Air^^^^^en|Jazz Aviation|=
 air-jeju-air-v1^^2005-01-01^^JJA^7C^806^Jeju Air^^^^^https://en.wikipedia.org/wiki/Jeju_Air^en|Jeju Air|^^air-jeju-air^1^
 air-jet2-v1^^2003-02-12^^EXS^LS^949^Jet2.com^^^^^https://en.wikipedia.org/wiki/Jet2.com^en|Jet2.com|^^air-jet2^1^
 air-jet-4-you-v1^^^^^8J^0^Línea Aerea Eco Jet^^^^^^en|Línea Aerea Eco Jet|^^air-jet-4-you^1^
+air-jetair-caribbean-v1^^2022-11-15^^4J^JRC^JetAir Caribbean^^^^^https://nl.wikipedia.org/wiki/Jetair_Caribbean^^^^1^
```
Comment (alexprengere) [inline opentraveldata/optd_airline_best_known_so_far.csv:562]: There are 2 issues with that line:
- there is a missing caret right before the airline name
- the 2-letter and 3-letter codes are swapped

**Comment by alexprengere** [comment]
Thanks! If you address my 2 comments I will then merge your PR.

**Comment by thepermanenturl** [comment]
Made the changes and Fixed my typos. Would you prefer if it was one commit ?

**Comment by alexprengere** [comment]
No need, as GitHub allows squashing when merging the PR. Thanks! I have regenerated the final airlines file as well.

**Pull request merged**